### ISSUE TITLE
In detailslist component of office ui fabric react, when we tab to each row,NVDA mention the word row twice

### ISSUE THREAD

**Issue by SriNandiniG** [nvaccess/nvda#9444]
### Steps to reproduce:
https://codepen.io/vitalius1/pen/OqerJN

### Actual behavior:
when we navigate to the rows in the fable, nvda mentions all the details in the table follow by row row 4(i.e, row number). for example, in the codepen given when we navigate to the first row nvda reads it as Lorem ipsum.vsdx Dolor Sit 62kb row row one

### Expected behavior:
NVDA should not mention the row twice for example: in the codepen given when we navigate to the first row nvda reads it as Lorem ipsum.vsdx Dolor Sit 62kb row one

#### NVDA installed/portable/running from source:
https://www.nvaccess.org/download/
#### NVDA version:
2018.4.1
#### Windows version:
Windows 10
#### Does the issue still occur after restarting your PC?
yes

**Comment by Adriani90** [comment]
@SriNandiniG are you able to reproduce this issue in NVDA 2019.2? In which browser and which version do you have this issue?

**Comment by Adriani90** [comment]
No update from the initial autor since more than one year, I cannot reproduce with NVDA 2020.1 in Firefox 77, Chrome 83 or MS Edge Chromium 83 or Internet explorer 11. I am closing as abandoned and works for me.
